Scientific_workflows/WIT_Metrics/wit_metrics.ipynb

```diff
@@ -0,0 +1,472 @@
+{
```

Comment:
<p>Maybe make this an <code>if</code>  instead of a raw cell.</p><p> </p><pre class="ql-syntax" spellcheck="false">load_from_db = False
if load_from_db:
  from wit_tooling import query_wit_data
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/772/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>